agreements/project-contribution/fiscal-sponsorship-amendment-email.md

```diff
@@ -0,0 +1,45 @@
+# Email Template for Amendment to Fiscal Sponsorship Agreement
+
+This email format can be used by a Project Representative or Signatory to request a change to the Fiscal Sponsorship Agreement (FSA).
```

Comment:
I get that the Fiscal Sponsorship Agreement is a living thing.  So I understand the amendment  use-case.